modules/entity/spec/unsorted_state_adapter.spec.ts

```diff
@@ -348,3 +349,198 @@ describe('Unsorted State Adapter', () => {
     });
   });
 });
+
+describe('Unsorted State Adapter with string entities', () => {
+  let adapter: EntityStateAdapter<TypeModel>;
+  let state: EntityState<TypeModel>;
+
+  beforeEach(() => {
+    adapter = createEntityAdapter({
+      selectId: (type: TypeModel) => type,
+    });
+
+    state = { ids: [], entities: {} };
+  });
+
+  it('should let you add one entity to the state', () => {
+    const withOneEntity = adapter.addOne(BookType, state);
+
+    expect(withOneEntity).toEqual({
+      ids: [BookType],
+      entities: {
+        [BookType]: BookType,
+      },
+    });
+  });
+
+  it('should not change state if you attempt to re-add an entity', () => {
+    const withOneEntity = adapter.addOne(BookType, state);
+
+    const readded = adapter.addOne(BookType, withOneEntity);
+
+    expect(readded).toBe(withOneEntity);
+  });
+
+  it('should let you add many entities to the state', () => {
+    const withOneEntity = adapter.addOne(MovieType, state);
+
+    const withManyMore = adapter.addMany([BookType, ShowType], withOneEntity);
+
+    expect(withManyMore).toEqual({
+      ids: [MovieType, BookType, ShowType],
+      entities: {
+        [MovieType]: MovieType,
```

Comment:
What's the purpose of it? From what I see you'd need to pass something, to get the same thing.

Reply:
testing if the addMany() works with primitives objects, or doesn't break in the future.